aws/data_source_aws_route53_resolver_endpoint.go

```diff
@@ -0,0 +1,178 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/route53resolver"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
hey @robh007 just a heads up this has now changed to `"github.com/hashicorp/terraform-plugin-sdk/helper/schema"`